### ISSUE TITLE
Set userInfo

### ISSUE THREAD

**Issue by SeveHo** [skrafft/react-native-jitsi-meet#95]
Hi,
it would be nice to be able to set the `builder.userInfo` so the solution for setting the displayName as described here: https://github.com/jitsi/jitsi-meet/issues/1880#issuecomment-552199143 would work.

You probably only have to extend `RNJitsiMeetViewManager.m`
```ObjectiveC
RCT_EXPORT_METHOD(call:(NSString *)urlString userInfo:(NSDictionary *)userInfo)
{
    RCTLogInfo(@"Load URL %@", urlString);
    dispatch_sync(dispatch_get_main_queue(), ^{
        JitsiMeetConferenceOptions *options = [JitsiMeetConferenceOptions fromBuilder:^(JitsiMeetConferenceOptionsBuilder *builder) {        
            builder.room = urlString;
            builder.userInfo = userInfo
        }];
        [jitsiMeetView join:options];
    });
}
``` 

Do you think you can do this in the close future? I'd rather like to use your package than having to create a forked npm package just for that.

Regards

**Comment by skrafft** [comment]
Hi @SeveHo,

Good idea. I'll probably do it this weekend. I will keep you posted when it's done.

**Comment by SeveHo** [comment]
Hi @skrafft ,

do you have any update about the request? Waiting wishfully for this.

**Comment by skrafft** [comment]
Hi @SeveHo,

Sorry, I've been busier than initially thought. I just published version 2.1.0 that adds this feature.
Feel free add your comments here.
